packages/extension-vscode/README.md

```diff
@@ -0,0 +1,27 @@
+# webhint extension for Visual Studio Code
+
+Use `webhint` to improve your website - during development.
```

Comment:
Maybe we should add a `gif` showing the extension at work?

Reply:
@alrra, do you have a favorite service/tool for converting videos to `gif` by chance?

Reply:
@antross No, I usually just create the `gif` directly. Maybe try https://ezgif.com/video-to-gif (I sometimes use that website to edit `gif`s, and so far, I am satisfied with it).

Reply:
@alrra That site worked well. Here's the `gif`. Let me know what you think. 
@molant You should take a look too.

![vscode-webhint](https://user-images.githubusercontent.com/785009/45783270-ef0c2900-bc2a-11e8-955f-9438cd93fba1.gif)

Reply:
😍

Reply:
@alrra Any concerns about file size if we're checking this into the repository? The `gif` is ~1.5MB. I'm wondering if we should host it elsewhere and link to it...

Reply:
> The gif is ~1.5MB

Can you share the original video with me? Thanks!

Reply:
@antross Open another PR when did is done. Thanks!